### DISCUSSION TITLE
Can organization have multiple owners

### DISCUSSION THREAD

**Discussion by Sgeete** [community/community#22893] [New to GitHub]
<p>Hi,</p>
<p>I am currently the owner of the organization. Can I add someone else as well as an owner while retaining me as an owner as well.</p>
<p>Thanks,</p>

**Comment by nadiajoyce** [comment, marked as answer]
<p>Yes! Take a look at this guide:</p>
<p><a href="https://help.github.com/articles/changing-a-person-s-role-to-owner/" rel="nofollow noopener">https://help.github.com/articles/changing-a-person-s-role-to-owner/</a></p>
